React/Modules/RCTAccessibilityManager.m

```diff
@@ -162,6 +163,16 @@ - (void)setMultipliers:(NSDictionary<NSString *, NSNumber *> *)multipliers
   self.multipliers = multipliers;
 }
 
+RCT_EXPORT_METHOD(setAccessibilityFocus:(nonnull NSNumber *)reactTag)
+{
+  dispatch_async(RCTGetUIManagerQueue(), ^{
+    [self.bridge.uiManager addUIBlock:^(__unused RCTUIManager *uiManager, NSDictionary<NSNumber *, UIView *> *viewRegistry) {
```

Comment:
@javache Sweet, I wasn't aware of `viewForReactTag:`.

I addressed your feedback.